site-docker.yml.sample

```diff
@@ -1,6 +1,9 @@
 ---
 # Defines deployment design and assigns role to server groups
 
+- hosts: all
```

Comment:
Do we need `all` in this case? Should we restrict it to just the `mons` or the `mons`, `osds`, `mdss`, `rgws`, `nfss`, and `restapis`?